android/README.md

```diff
@@ -15,33 +15,25 @@ If you are interested in the iOS version, you can find more information about it
 ## Use the project with your own Firebase instance
 
 1. Clone this repository.
-
-2. Create `signing.properties` file in your `app` folder with the following contents:
-
-    ```
-    storeFile=~/android/debug.keystore  # path to your debug.keystore
-    storePassword=android
-    keyAlias=androiddebugkey
-    keyPassword=android
-    ```
-
-3. Create a new project in the [Firebase console][3].
-
-4. Click *Add Firebase to your Android app*
+- Create a new project in the [Firebase console][3].
+- Click *Add Firebase to your Android app*
   * provide a **unique package name**
   * use the same package name for the **applicationId** in your `build.gradle`
   * insert SHA-1 fingerprint of your debug certificate, otherwise you won't be able to log in
+- Copy the generated *google-services.json* to the `app` folder of your project.
+- You should be able to successfully sync the project now.
+- Copy contents of the `../server/database.rules.json` into your *Firebase -> Database -> Rules* and publish them.
+- Enable **Google sign-in** in your *Firebase -> Auth -> Sign-in Method*.
+- Build and run the app.
 
-5. Copy the generated *google-services.json* to the `app` folder of your project.
+#### Creating a signed release apk
 
-6. You should be able to successfully sync the project now.
-
-6. Copy contents of the `../server/database.rules.json` into your *Firebase -> Database -> Rules* and publish them.
-
-7. Enable **Google sign-in** in your *Firebase -> Auth -> Sign-in Method*.
-
-8. Build and run the app.
+This step is optional. To build a signed release apk, create a `signing.properties` file in your `app` folder with the following content:
 
+    storeFile=~/android/debug.keystore  # path to your debug.keystore
```

Comment:
Maybe instead of mentioning the debug keystore and it's credentials just mention the keystore as a generic thing.
We don't want to encourage people using `debug.keystore` when we are talking about a **release** build.